src/components/TableOfContents.tsx

```diff
@@ -322,50 +227,63 @@ const TableOfContentsMobile: React.FC<IPropsTableOfContentsMobile> = ({
   maxDepth,
   className,
 }) => {
-  const [isOpen, setIsOpen] = useState<boolean>(false)
+  // const [isOpen, setIsOpen] = useState<boolean>(false)
+  const { getButtonProps, getDisclosureProps, isOpen } = useDisclosure({
+    defaultIsOpen: false,
+  })
   if (!items) {
     return null
   }
 
   return (
-    <AsideMobile className={className}>
-      <HeaderMobile onClick={() => setIsOpen(!isOpen)}>
-        <HeaderText>
+    <Box
+      as="aside"
+      background="background"
+      border="1px"
+      borderColor="border"
+      borderRadius="4px"
+      py={2}
+      px={4}
+      /* TODO find better way - this accounts for huge header margin top */
+      /* but if doc DOESN'T start w/ header, it overlaps, e.g. /docs/accounts/  */
+      /* margin-bottom: -10rem; */
+      className={className}
+    >
+      <Flex
+        color="text200"
+        cursor="pointer"
+        alignItems="center"
+        justify="space-between"
+        {...getButtonProps()}
+      >
+        <chakra.span flex={1} fontWeight={500}>
           <Translation id="on-this-page" />
-        </HeaderText>
-        <IconContainer
-          variants={dropdownIconContainerVariant}
-          animate={isOpen ? "open" : "closed"}
-        >
-          <MobileIcon name="chevronDown" />
-        </IconContainer>
-      </HeaderMobile>
-      <OuterList
-        animate={isOpen ? "open" : "closed"}
-        variants={{
-          open: {
-            opacity: 1,
-            display: "block",
-            transition: {
-              duration: 0.6,
-            },
-          },
-          closed: {
-            opacity: 0,
-            transitionEnd: {
-              display: "none",
-            },
-          },
-        }}
-        initial="closed"
+        </chakra.span>
+        <Icon
+          as={MdExpandMore}
+          transform={isOpen ? "rotate(0)" : "rotate(-90deg)"}
+          boxSize={6}
+          transition="transform .4s"
+        />
+      </Flex>
+      <Fade
+        in={isOpen}
+        {...getDisclosureProps()}
+        transition={{ enter: { duration: 0.6 } }}
```

Comment:
Removing the import of `motion` in favor of Chakra's `Fade` component. Since disclosure props are added to increase accessibility, collapse animation is different on close, though the transition duration is retained.